pubsub/README.md

```diff
@@ -10,13 +10,66 @@ Google Cloud Platform pub/sub library.
 
 ## Installation
 
-```
+```toml
 [dependencies]
 google-cloud-pubsub = <version>
 google-cloud-default = { version = <version>, features = ["pubsub"] }
 ```
 
-## Quick Start
+## Quickstart
+
+### Authentication
+
+When you are not using an emulator you'll need to be authenticated.
+There are two ways to do that:
+
+#### Automatically
+You can use [google-cloud-default](https://crates.io/crates/google-cloud-default) to create [ClientConfig](https://docs.rs/google-cloud-pubsub/0.14.0/google_cloud_pubsub/client/struct.ClientConfig.html)
+
+This will try and read the credentials from a file specified in the environment variable `GOOGLE_APPLICATION_CREDENTIALS_JSON` or
```

Comment:
To be precise, it reads from `GOOGLE_APPLICATION_CREDENTIALS` or `GOOGLE_APPLICATION_CREDENTIALS_JSON` or from a metadata server.

Reply:
added it